packages/react/package.json

```diff
@@ -128,6 +128,7 @@
     "@jupyterlab/translation-extension": "4.0.3",
     "@jupyterlab/ui-components-extension": "4.0.3",
     "@lumino/default-theme": "2.1.2",
+    "@primer/react": "0.0.0-20230912151940",
```

Comment:
primer/react is already pull via transitive dependency by "@datalayer/primer-addons": "0.2.0",

Reply:
Do you added when it becomes a direct dependency as here in the theme.ts file?